### ISSUE TITLE
Incompatible with ido-use-virtual-buffers

### ISSUE THREAD

**Issue by purcell** [nex3/perspective-el#10]
In the latest Emacs, `ido-use-virtual-buffers` enables the re-opening of recently-edited files by including them in the buffer list displayed by `ido-switch-buffer`.  In persp-mode, those buffers don't get listed.  I'm not exactly sure what the ideal behavior would be in this case, but perhaps it would be useful to add a toggle which would allow persp-mode to honor that ido flag.

**Comment by nex3** [comment]
I don't use `ido` personally but I'd love to get a patch for this.

**Comment by zenspider** [comment]
This is ~8 years old with no real new comments... is this still relevant? OP? Can you close if not?

**Comment by thisch** [comment]
Yes, this is still relevant, although I don't use ido but ivy, which has a similar flag `ivy-use-virtual-buffers`.   
It should be possible to honor this flag in the new `persp-ivy-switch-buffer` defun.

**Comment by gcv** [comment]
Very interesting. Clearly not a feature of Ido or Ivy I've looked at before. I'd like some input from people who use virtual buffers in either framework. What behavior would make the most sense?

1. Present closed buffers from any past or present perspective (the easiest and most efficient solution, but possibly confusing).

2. Only present closed buffers which were formerly in the current perspective. If so: (1) What should happen when the perspective is closed? (2) Should closed buffer history be retained if a new perspective is opened with the same name? (3) When a buffer is removed from a perspective, should it stay in the virtual list?
